Dockerfile

```diff
@@ -1,38 +1,52 @@
-ARG VERSION=unspecified
+ARG GIT_COMMIT=unspecified
+ARG GIT_REMOTE=unspecified
+ARG VERSION=0.0.1
 
-FROM python:3.9.6-alpine
+FROM python:3.9.6
 
+ARG GIT_COMMIT
+ARG GIT_REMOTE
 ARG VERSION
 
 # For a list of pre-defined annotation keys and value types see:
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
 # Note: Additional labels are added by the build workflow.
+LABEL git_commit=$GIT_COMMIT
+LABEL git_remote=$GIT_REMOTE
 LABEL org.opencontainers.image.authors="mark.feldhousen@cisa.dhs.gov"
 LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security Agency"
+LABEL version=$VERSION
 
 ARG CISA_UID=421
 ENV CISA_HOME="/home/cisa"
+ENV PCA_REPORT_TOOLS_SRC="/usr/src/pca-report-tools"
 ENV ECHO_MESSAGE="Hello World from Dockerfile"
 
 RUN addgroup --system --gid ${CISA_UID} cisa \
   && adduser --system --uid ${CISA_UID} --ingroup cisa cisa
 
-RUN apk --update --no-cache add \
-ca-certificates \
-openssl \
-py-pip
+RUN apt-get install wget
 
-WORKDIR ${CISA_HOME}
+RUN apt-get update && \
+ apt-get install --no-install-recommends -y texlive texlive-xetex texlive-bibtex-extra
```

Comment:
I have added this as part of the acceptance criteria in issue https://github.com/cisagov/pca-report-generator-docker/issues/11

Reply:
Since `apk` is not being used now (see #11), this is partially OBE.  The equivalent change with `apt-get` was taken care of in https://github.com/cisagov/pca-report-generator-docker/pull/10/commits/ebc31822cf39304fed84fa7b0e3ecbe8777a113c.